packages/cashier/src/pages/withdrawal/withdrawal-crypto-receipt/withdrawal-crypto-receipt.tsx

```diff
@@ -1,41 +1,68 @@
 import React from 'react';
 import { Button, Clipboard, Icon, Text } from '@deriv/components';
-import { isMobile } from '@deriv/shared';
+import { useCryptoTransactions } from '@deriv/hooks';
+import { TModifiedTransaction } from '@deriv/hooks/src/useCryptoTransactions';
 import { observer, useStore } from '@deriv/stores';
 import { Localize, localize } from '@deriv/translations';
-import { useCashierStore } from '../../../stores/useCashierStores';
-import { TAccount } from '../../../types';
-import { getAccountText } from '../../../utils/utility';
+import { useCashierStore } from 'Stores/useCashierStores';
+import { TAccount } from 'Types';
+import { getAccountText } from 'Utils/utility';
+import { cryptoTransactionMapper } from '../../../modules/transactions-crypto/helpers';
 import './withdrawal-crypto-receipt.scss';
 
 type TWalletInformationProps = {
     account: TAccount;
     blockchain_address: string;
+    is_mobile: boolean;
 };
 
-const Status = () => {
+type TStatusProps = {
+    last_transaction:
+        | (TModifiedTransaction & {
+              is_deposit: boolean;
+              is_withdrawal: boolean;
+          })
+        | undefined;
+};
+
+const Status = ({ last_transaction }: TStatusProps) => {
+    const [status, setStatus] = React.useState({ status_color: 'warning', status_name: 'In review' });
+
+    const colorMap: { [key: string]: string } = {
+        warning: 'orange',
+        unsuccessful: 'red',
+        default: 'green',
```

Comment:
Can you please check if we can do something like this then pass it in `custom_color` prop of Icon component?
```suggestion
        warning: 'var(--status-warning)',
        unsuccessful: 'var(--status-danger)',
        default: 'var(--status-success)',
```

Reply:
sure 👍

Reply:
just checked, so we can't use the above and i need to pass in the direct color string to `Icon` component. This is because in the Icon component, color is changed based off these following classNames where it checks for green, red, and orange

![Screenshot 2023-11-16 at 2 07 12 PM](https://github.com/binary-com/deriv-app/assets/103412909/d082dd04-4680-4283-99e0-6ecba8aab269)